scripts/DeploySGXTEEVerifier.s.sol

```
@@ -0,0 +1,32 @@
+import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {EspressoSGXTEEVerifier} from "src/EspressoSGXTEEVerifier.sol";
+import {IEspressoSGXTEEVerifier} from "src/interface/IEspressoSGXTEEVerifier.sol";
+
+contract DeploySGXTEEVerifier is Script {
+    bytes32 constant SGX_VERIFIER_SALT = keccak256("espresso.sgxverifier.v1");
+
+    function run() external {
+        vm.startBroadcast();
+        bytes32 enclaveHash = vm.envBytes32("SGX_ENCLAVE_HASH");
+        address quoteVerifierAddr = vm.envAddress("SGX_QUOTE_VERIFIER_ADDRESS");
```

should we throw errors if env not set?

---

good idea

---

https://github.com/EspressoSystems/espresso-tee-contracts/pull/8/commits/c975bb4fb44ed9dcc92a5438c861eeb07dd1d7fc